### ISSUE TITLE
When moving columns, then removing one - last one gets lost

### ISSUE THREAD

**Issue by cyvvilek84** [handsontable/handsontable#5196]
I'm tring to dynamically remove some columns from Handsontable but can't really figure it out why after moving some columns around it will loose the last one.

After each columns move I update my local settings with recent headers order and sort the columns same way:

```
hotSettings.colHeaders = [hot.getColHeader(0), hot.getColHeader(1), 
                          hot.getColHeader(2), hot.getColHeader(3), 
                          hot.getColHeader(4), hot.getColHeader(5)];
hotSettings.columns.sort((col1, col2) =>
  hotSettings.colHeaders.indexOf(col1.data) - hotSettings.colHeaders.indexOf(col2.data);
});
```
Then on click I remove first element from both of those and update the Handsontable:

```
hotSettings.columns = hotSettings.columns.slice(1);
hotSettings.colHeaders = hotSettings.colHeaders.slice(1);

hot.updateSettings(hotSettings);
```

### Steps to reproduce

1. grab any column and move it to the last position
2. click 'remove column' button

### Demo

http://jsfiddle.net/spd8y4hm/2/

Some of the stuff intentionally hardcoded for simplicity

**Comment by AMBudnik** [comment]
This issue should be fixed with https://github.com/handsontable/handsontable/issues/5112
I have simplified the demo a little bit and tested at http://jsfiddle.net/8sgyn0p7/ and it works OK when we just delete the last column but after moving the columns I get the same issue as you did.

**Comment by wojciechczerniak** [comment]
Duplicate of #5200

**Comment by AMBudnik** [comment]
Referenced issue https://github.com/handsontable/handsontable/issues/5200 fixed in v8.0.0.
